### ISSUE TITLE
bug: npm registry https://registrynpm.storefrontcloud.io is returning a 401

### ISSUE THREAD

**Issue by kamranayub** [vuestorefront/storefront-next13-boilerplate#75]
### Environment

Environment: Using GitHub codespaces

### Reproduction

To reproduce, the easiest way is to click "Use this template" and "Open in Codespaces", then try to run `yarn install`.

![image](https://github.com/vuestorefront/storefront-next13-boilerplate/assets/563819/7408c26e-27a7-4416-b4cf-2ec1900098fb)

### Describe the bug

When trying to run `yarn install`, it appears the custom registry for Storefront doesn't allow public downloads and is returning a 401 Unauthorized error.

### Logs

```shell-script
error An unexpected error occurred: "https://registrynpm.storefrontcloud.io/@vsf-enterprise%2fsdk/-/sdk-1.0.0.tgz: Request failed \"401 Unauthorized\"".
```
```

**Comment by lukasborawski** [comment]
@kamranayub Issue will be fixed with #76. Thanks for sharing.

**Comment by lukasborawski** [comment]
Done.
